apps/full-site-editing/full-site-editing-plugin/global-styles/src/global-styles-sidebar.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import { PluginSidebar } from '@wordpress/edit-post';
+import { Button, PanelBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+
+/**
+ * Internal dependencies
+ */
+import FontPairingsPanel from './font-pairings-panel';
+import FontSelectionPanel from './font-selection-panel';
+import { GlobalStylesIcon } from './icon';
+import { FONT_BASE, FONT_HEADINGS } from './constants';
+
+const ANY_PROPERTY = 'ANY_PROPERTY';
+
+const isFor = filterProperty => option =>
+	option.prop === ANY_PROPERTY || option.prop === filterProperty;
+
+const toOption = font => {
+	if ( typeof font === 'object' ) {
+		const { label, value, prop = ANY_PROPERTY } = font;
+		return { label, value, prop };
+	}
+	return { label: font, value: font, prop: ANY_PROPERTY };
+};
+const isNotNull = option => option.value !== null && option.label !== null;
+
+const toOptions = ( options, filterProperty ) =>
+	! options
+		? []
+		: options
+				.map( toOption )
+				.filter( isNotNull )
+				.filter( isFor( filterProperty ) );
+
+const PanelActionButtons = ( { hasLocalChanges, resetAction, publishAction, className } ) => (
+	<div className={ className }>
+		<Button disabled={ ! hasLocalChanges } isDefault onClick={ resetAction }>
+			{ __( 'Reset' ) }
+		</Button>
+		<Button
+			disabled={ ! hasLocalChanges }
+			style={ { marginLeft: '1em' } }
```

Comment:
This inline style should likely be moved to CSS class selector.

Reply:
Extracted at https://github.com/Automattic/wp-calypso/pull/36288/commits/5386b874f5bf77b1ca20fbe6446f7c53c9b69ca3